src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritConnection.java

```diff
@@ -94,6 +94,7 @@
     private AuthenticationUpdater authenticationUpdater = null;
     private final Set<ConnectionListener> listeners = new CopyOnWriteArraySet<ConnectionListener>();
     private int sshRxBufferSize = SSH_RX_BUFFER_SIZE;
+    private StringBuilder eventBuffer = null;
```

Comment:
Actually when you have configured your Jenkins against 2 Gerrit instances you have 2 GerritConnection threads running.
But for the StringBuilder this is not a problem. It is an instance variable and the only place where it is created, modified or deleted is in the private getLine method. So no problem with multiple threads.